src/components/wizard-internet/wizard-internet.html

```diff
@@ -98,155 +99,80 @@
-                  <span class="fa form-control-feedback"
-                        ng-class="formControlFeedback('vpnKey')"></span>
-                  <div class="help-block">
-                    <div ng-messages="hasError('vpnKey')">
-                      <div ng-message="pattern">
-                        <span translate>.key.error.pattern</span>
-                      </div>
-                    </div>
-                  </div>
-                </div>
+            <div class="row">
+              <div ng-repeat="config in $ctrl.internetTunnelConfSecret">
+                <wizard-upload-configfile details-string="{{config}}"
+                                          on-update="$ctrl.newInternet.internetTunnel.files[config.property] = fileContent"
+                                          file-content="$ctrl.newInternet.internetTunnel.files[config.property]"
+                ></wizard-upload-configfile>
               </div>
             </div>
-            <div class="col-md-6">
-              <div class="form-group has-feedback"
-                   ng-class="formFeedback('taKey')">
-                <label for="taKey" class="col-md-5 control-label" translate>
-                  .takey.label
-                </label>
-                <div class="col-md-7">
-                  <div class="btn btn-info" role="button" id="taKey" name="taKey"
-                       data-ng-model="taKey" data-ngf-select="uploadVpnFiles($files, 'takey')"
-                       data-ngf-multiple="false" ngf-pattern="'text/*,.key'" ngf-accept="'text/*,.key'"
-                       translate>
-                    .takey.select
-                  </div><br/>&nbsp;{{wizard.internet.vpn.takey.filename}}
-                  <span class="fa form-control-feedback"
-                        ng-class="formControlFeedback('taKey')"></span>
-                  <div class="help-block">
-                    <div ng-messages="hasError('taKey')">
-                      <div ng-message="pattern">
-                        <span translate>.takey.error.pattern</span>
-                      </div>
-                    </div>
-                  </div>
-                </div>
+            <div class="row">
+              <div class="col-md-offset-9 col-md-3">
+                <span class="btn-danger btn-sm" ng-click="$ctrl.clearFiles()">
+                  <i class="fa fa-trash-o" aria-hidden="true"></i>
+                  <span translate>
+                    .clearAllFiles
+                  </span>
+                </span>
               </div>
             </div>
           </div>
         </div>
         <div class="row" translate-namespace=".vpninfo">
           <div class="col-md-offset-3 col-md-9">
-            <div ng-click="toggleVpnList()">
-              <span ng-show="!state.internet.showVpnList">&#9658;</span>
-              <span ng-show="state.internet.showVpnList">&#9660;</span>
+            <div ng-click="$ctrl.toggleVpnList()">
```

Comment:
I think we should just link to a wiki page and remove the list from the wizard. The wiki page can also go into more detail.